gno.land/cmd/gnoland/start_test.go

```diff
@@ -1,56 +0,0 @@
-package main
-
-import (
-	"bytes"
-	"context"
-	"path/filepath"
-	"testing"
-	"time"
-
-	"github.com/gnolang/gno/tm2/pkg/commands"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestStartInitialize(t *testing.T) {
```

Comment:
Please confirm that lazy initialization won't cause a panic. Move it to pkg/gnoland from cmd; it can be straightforward. We are checking for possible runtime panics.

Reply:
I've added support for this lazy init testing here:
[d5e3f70](https://github.com/gnolang/gno/pull/1985/commits/d5e3f70aa83e03fff67222e3061d0af697fcd1bf)

Not just that, but now we can run any kind of scenario for the `gnoland start` command, since I've cleaned up how we manage the command context